networkit/cpp/centrality/test/DynBetweennessGTest.cpp

```diff
@@ -112,6 +131,79 @@ TEST_F(DynBetweennessGTest, runDynApproxBetweennessSmallGraphEdgeDeletion) {
 
 }
 
+TEST_P(DynBetweennessGTest, testDynApproxBetweenessGeneratedGraph) {
+    ErdosRenyiGenerator generator(100, 0.25, isDirected());
+    Graph G1 = generator.generate();
+    Graph G(G1, isWeighted(), isDirected());
```

Comment:
Why are we creating two graphs while we only need one? If you need a weighted graph you can use `GraphTools::toWeighted`.